packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -817,6 +817,21 @@ export class DeltaManager
         let listenerToClear: ((op: ISequencedDocumentMessage) => void) | undefined;
 
         if (to !== undefined) {
+            const lastExpectedOp = to - 1; // make it inclusive!
+
+            // It is possible that due to asynchrony (including await above), required ops were already
+            // received through delta stream. Validate that before moving forward.
+            if (this.lastProcessedSequenceNumber >= lastExpectedOp) {
+                this.logger.sendPerformanceEvent({
+                    reason: this.fetchReason,
+                    eventName: "ExtraStorageCallEarly",
```

Comment:
What is meant by "ExtraStorageCallEarly"?

Reply:
I'll rename it to "ExtraStorageCall" and add property to differentiate against another instance - take a look at comment above it.